README.md

```diff
@@ -9,7 +9,7 @@
 
 > OpenPGP implemented in pure Rust, permissively licensed
 
-rPGP is the only full Rust implementation of OpenPGP, following [RFC4880](https://tools.ietf.org/html/rfc4880.html) and [RFC2440](https://tools.ietf.org/html/rfc2440). It offers a minimal low-level API and does not prescribe trust schemes or key management policies. It fully supports all functionality required by the [Autocrypt 1.1 e-mail encryption specification](https://autocrypt.org/level1.html).
+rPGP is the only pure Rust implementation of OpenPGP, following [RFC4880](https://tools.ietf.org/html/rfc4880.html) and [RFC2440](https://tools.ietf.org/html/rfc2440). It offers a minimal low-level API and does not prescribe trust schemes or key management policies. It fully supports all functionality required by the [Autocrypt 1.1 e-mail encryption specification](https://autocrypt.org/level1.html).
```

Comment:
Would you agree that when using the rust crypto backend, Sequoia is also "pure Rust".  If so, perhaps drop "only".

Reply:
This was written many moons ago originally :) I honestly don't know, as I have only heard last week about sequoia having a rust backend.

Reply:
To be honest I tried cross-compiling Sequoia yesterday and I think at least bzip2 compression requires native libraries (correct me if I'm wrong). Not sure how rPGP handles that.

Reply:
CI cross compiles for various androids and wasm: https://github.com/rpgp/rpgp/actions/runs/4566213673

I am going to merge it as is for now, but am happy to update/accept PRs if there is more info on this.

Reply:
> To be honest I tried cross-compiling Sequoia yesterday and I think at least bzip2 compression requires native libraries (correct me if I'm wrong). Not sure how rPGP handles that.

Looking [at the dependencies](https://github.com/rpgp/rpgp/blob/master/Cargo.toml#L70), it looks like rpgp supports flate, but not bzip2.  @dignifiedquire is that correct?

Reply:
yep, zip is supported but not bzip2 I think that was the reason because I didn’t find a pure rust version

Reply:
Thanks for the clarification.

Reply:
Yes, it seems there are no good quality, pure Rust versions: https://crates.io/search?q=bzip2

There is one but it doesn't look like it is maintained: https://crates.io/crates/bzip2-rs